pkgs/kde/third-party/krohnkite/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nodejs_22,
+  kwin,
+  kpackage,
+  zip,
+}:
+buildNpmPackage rec {
+  pname = "krohnkite";
+  version = "0.9.7";
+
+  src = fetchFromGitHub {
+    owner = "anametologin";
+    repo = "krohnkite";
+    rev = "refs/tags/${version}";
+    hash = "sha256-8A3zW5tK8jK9fSxYx28b8uXGsvxEoUYybU0GaMD2LNw=";
+  };
+
+  npmDepsHash = "sha256-My1goFEoZW9kFA3zb8xKPxAPXm6bypyq+ajPM8zVOHQ=";
+
+  dontWrapQtApps = true;
+
+  nodejs = nodejs_22;
+
+  nativeBuildInputs = [
+    kpackage
+    zip
+    kwin
+  ];
+
+  postPatch = ''
+    cp ${./package-lock.json} package-lock.json
```

Comment:
Have you asked upstream if it would be possible to add a lockfile?
It would be nice if we could drop this vendored lockfile

Reply:
It's fine. If we convince upstream until the next release to add one we take it, otherwise we just keep generating one ourselves.